src/OpenTelemetry.Api/README.md

```diff
@@ -85,26 +85,25 @@ OpenTelemetry SDK ships a BaggagePropagator and enables it by default.
 
 ```csharp
 // Use Baggage.Current to get all the key/value pairs present in Baggage
-foreach (var item in Baggage.Current)
+foreach (var item in Baggage.GetBaggage())
```

Comment:
the line above has a comment referring to Baggage.Current.

Reply:
Fixed